lib/plugins/aws/metrics/awsMetrics.js

```diff
@@ -0,0 +1,209 @@
+'use strict';
+
+const BbPromise = require('bluebird');
+const chalk = require('chalk');
+const _ = require('lodash');
+const moment = require('moment');
+const validate = require('../lib/validate');
+
+class AwsMetrics {
+  constructor(serverless, options) {
+    this.serverless = serverless;
+    this.options = options;
+    this.provider = this.serverless.getProvider('aws');
+
+    Object.assign(this, validate);
+
+    this.hooks = {
+      'metrics:metrics': () => BbPromise.bind(this)
+        .then(this.extendedValidate)
+        .then(this.getMetrics)
+        .then(this.showMetrics),
+    };
+  }
+
+  extendedValidate() {
+    this.validate();
+
+    // validate function exists in service
+    this.options.function = this.serverless.service.getFunction(this.options.function).name;
+
+    const today = new Date();
+    let yesterday = new Date();
+    yesterday = yesterday.setDate(yesterday.getDate() - 1);
+    yesterday = new Date(yesterday);
+
+    if (this.options.startTime) {
+      const since = (['m', 'h', 'd']
```

Comment:
Hmm. I modified it a little bit. The code which is used in the `metrics` plugin does a little less than the one in the `logs` plugin.